boards/arm/nucleo_l073rz/doc/index.rst

```diff
@@ -59,6 +59,8 @@ Nucleo L073RZ provides the following hardware components:
   - Mass storage
   - Debug port
 
+- True random number generator
```

Comment:
This is the list of Hardware component provided by the board.
But RNG is not provided by the board  (rather by the MCU)
so there is no reason to add RNG in this list